### PR TITLE
Add instructions for local debugging

### PR DESCRIPTION
**Pull request by stew-r**
Adds steps to attach a debugger to your function running locally.

Addresses:

- https://github.com/GoogleCloudPlatform/functions-framework-nodejs/issues/47
- https://github.com/GoogleCloudPlatform/functions-framework-nodejs/issues/15

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -5,4 +5,35 @@ This directory contains advanced docs around the Functions Framework.
 - TODO: Run Multiple Cloud Functions [#23](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/issues/23)
 - TODO: Pub/Sub Trigger [#37](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/issues/37)
 - TODO: Deploy to Cloud Run [#28](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/pull/28)
-- TODO: Local Debug [#15](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/issues/15)
+
+## Debugging functions
+
+The Functions Framework works with standard tooling that you might use when writing a function for a Node.js environment. You can attach a debugger to your function by following these steps.
+
+1. Write a Node.js function:
```
Comment (grant) [inline docs/README.md]: This should be more instructional, like:

1. Create an `index.js` file:

File: docs/README.md [hunk 2]
```diff
@@ -5,4 +5,35 @@ This directory contains advanced docs around the Functions Framework.
...
+};
+```
+
+2. Install the Functions Framework:
+
+```sh
+npm install @google-cloud/functions-framework
+```
+
+3. Launch the Functions Framework and attach a debugger:
```
Comment (grant) [inline docs/README.md]: Nit: It's more like "Run the FF" rather than "Launch". We're dispatching a Node process.

File: docs/README.md (hunk 1 above)
Comment (stew-r) [reply docs/README.md]: Updated.

File: docs/README.md (hunk 2 above)
Comment (stew-r) [reply docs/README.md]: Updated.

File: docs/README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ exports.helloWorld = (req, res) => {
 npm install @google-cloud/functions-framework
 ```
 
-3. Launch the Functions Framework and attach a debugger:
+3. Run `node`, enable the inspector and run the Functions Framework (specifying the your target function):
```
Comment (grant) [inline docs/README.md]: Nit: I'd say this, which is simpler:

Expected:
3. Run `node`, enable the inspector, and run the Functions Framework:
Actual:
3. Run `node`, enable the inspector and run the Functions Framework (specifying the your target function):

Comment (stew-r) [reply docs/README.md]: Done

**Comment by grant** [comment]
Nice! Thanks for the contribution!

**Pull request merged**